PF/Assignment1/Answer.java

```diff
@@ -0,0 +1,40 @@
+//program to strore answer of user and increment customer number
+
+public class Answer {
+	int single[] = new int[10]; // for single select type
```

Comment:
Action: You have used static array. It will not work when count is greater than 10.